packages/server-core/src/setting/aws-setting/aws-setting.seed.ts

```diff
@@ -50,12 +46,18 @@ export async function seed(knex: Knex): Promise<void> {
           }
         }),
         s3: JSON.stringify({
+          accessKeyId: process.env.STORAGE_AWS_ACCESS_KEY_ID,
+          secretAccessKey: process.env.STORAGE_AWS_ACCESS_KEY_SECRET,
           endpoint: process.env.STORAGE_S3_ENDPOINT,
           staticResourceBucket: process.env.STORAGE_S3_STATIC_RESOURCE_BUCKET,
           region: process.env.STORAGE_S3_REGION,
           avatarDir: process.env.STORAGE_S3_AVATAR_DIRECTORY,
           s3DevMode: process.env.STORAGE_S3_DEV_MODE
         }),
+        eks: JSON.stringify({
```

Comment:
This will work for `dev-reinit` and for `prepare-database` when there is no existing record. But for scenarios where there is already an existing data and you used `prepare-database` then you will need to add this data to the column, else it will be empty.

You can have a look of `packages/server-core/src/setting/client-setting/client-setting.seed.ts` and see around line 69.

Reply:
I've added a migration which will perform this conversion.